include/RAJA/policy/openmp_target/forall.hpp

```diff
@@ -68,26 +68,30 @@ RAJA_INLINE void forall_impl(const omp_target_parallel_for_exec<ThreadsPerTeam>&
     ib(begin_it[i]);
   }
 
+  return resources::EventProxy<resources::Omp>(&res);
 }
 
 template <typename Iterable, typename Func>
-RAJA_INLINE void forall_impl(const omp_target_parallel_for_exec_nt&,
-                             Iterable&& iter,
-                             Func&& loop_body)
+RAJA_INLINE resources::EventProxy<resources::Omp> forall_impl(resources::Resource &omp_res,
+                                                              const omp_target_parallel_for_exec_nt&,
+                                                              Iterable&& iter,
+                                                              Func&& loop_body)
 {
   using Body = typename std::remove_reference<decltype(loop_body)>::type;
   Body body = loop_body;
 
   RAJA_EXTRACT_BED_IT(iter);
 
 #pragma omp target teams distribute parallel for schedule(static, 1) \
-    firstprivate(body,begin_it)
+    firstprivate(body,begin_it) \
+    depend(inout : omp_ptr)
```

Comment:
Where is omp_ptr?

Reply:
This is left over from trying to work through supporting omp target w/ forall, I'll remove it.

Reply:
Done.